sublime-text-merlin.py

```diff
@@ -369,7 +369,7 @@ def on_query_completions(self, view, prefix, locations):
             for r in result['entries']:
                 name = clean_whitespace(r['name'])
                 desc = clean_whitespace(r['desc'])
-                self.cplns.append(((name + '\t' + desc), name))
+                self.cplns.append(((name + '\t' + desc), make_snippet(name, desc)))
```

Comment:
One option that could work well is to store the last completion in a variable and add a keyboard shortcut to insert parameter snippets for the last completion. So you complete, then if you don't know the parameters you can type `cmd+h` or something to complete them.